src/styles/app.scss

```diff
@@ -4,11 +4,6 @@ html, body, #root {
   width: 100%;
 }
 
-#root {
-  overflow-x: hidden;
-  overflow-y: scroll;
```

Comment:
Same question re: scrolling as in other PR

Reply:
i don't get you point @bent0b0x, we need this overflow fix?

Reply:
We did at some point, im just making sure we’ve tested and made sure that it is no longer necessary

Reply:
...and it could causes a "double scroll"
![image](https://user-images.githubusercontent.com/320157/37529725-9cb90f44-2916-11e8-931f-a84c56809ae7.png)

The header its above the scroll because of the `position: fixed`